redbot/core/core_commands.py

```diff
@@ -2658,10 +2658,10 @@ async def license_info_command(ctx):
     """
 
     message = (
-        "This bot is an instance of Red-DiscordBot (hereafter referred to as Red)\n"
+        "This bot is an instance of Red-DiscordBot (hereafter referred to as Red).\n"
```

Comment:
I know it's not part of PR, but shouldn't `hereafter` be `hereinafter`?

Reply:
Yes, looking at that I think it should be.